editor/plugins/path_3d_editor_plugin.cpp

```diff
@@ -284,6 +283,24 @@ void Path3DGizmo::redraw() {
 		return;
 	}
 
+	debug_material = gizmo_plugin->get_material("path_material", this);
+
+	Color path_color = path->get_debug_custom_color();
+	if (path_color != Color(0.0, 0.0, 0.0)) {
+		debug_material.instantiate();
+		debug_material->set_albedo(path_color);
+		debug_material->set_shading_mode(StandardMaterial3D::SHADING_MODE_UNSHADED);
+		debug_material->set_transparency(StandardMaterial3D::TRANSPARENCY_ALPHA);
+		debug_material->set_flag(StandardMaterial3D::FLAG_SRGB_VERTEX_COLOR, true);
+		debug_material->set_flag(StandardMaterial3D::FLAG_ALBEDO_FROM_VERTEX_COLOR, true);
+		debug_material->set_flag(StandardMaterial3D::FLAG_DISABLE_FOG, true);
+	}
+
+	// 0. Don't draw since nothing will be visible anyway.
+	if (path_color.a == 0.0) {
+		return;
+	}
```

Comment:
The only thing I noticed is that if you set alpha to 0, the Path3D gizmo handles in the editor will disappear. I suggest removing this particular `return;` and only avoid creating the path, but not the handles further below.